models/agent_model.go

```diff
@@ -74,6 +75,8 @@ type Agent struct {
 	QueryExamplesDisabled bool    `reform:"query_examples_disabled"`
 	MaxQueryLogSize       int64   `reform:"max_query_log_size"`
 	MetricsURL            *string `reform:"metrics_url"`
+	AWSAccessKeyID        string  `reform:"aws_access_key_id"`
+	AWSSecretAccessKey    string  `reform:"aws_secret_access_key"`
```

Comment:
I would re-use username and password.
@BupycHuk WDYT?

Reply:
I think it's better to use as is to not confuse people. If we will use username and password people will think that they should use username and password for database.